edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/logs/LogMessageParser.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core.Logs
+{
+    extern alias akka;
+    using System;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using akka::Akka.IO;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    // Parses logs into message objects.
+    //
+    // Expected format -
+    // Each input payload should contain one frame in Docker format -
+    //    01 00 00 00 00 00 00 1f 52 6f 73 65 73 20 61 72  65 ...
+    //    │  ─────┬── ─────┬─────  R o  s e  s a  r e...
+    //    │       │        │
+    //    └stdout │        │
+    //            │        └─ 0x0000001f = 31 bytes (including the \n at the end)
+    //         unused
+    //
+    // The payload itself is expected to be in this format -
+    // <logLevel> [TimeStamp] log text
```

Comment:
do we still have square backet around timestamp? I saw there is another PR to update log time format.

Reply:
No it is not, just wanted to demark the Timestamp.